### PR TITLE
Add CI auto injection custom value to build scans

### PR DESCRIPTION
**Pull request by sorin-florea**
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

### CODE REVIEW

**Review by welandaz — APPROVED** [review]

**Comment by lemeurherve** [comment]
I would remove the .DS_Store file of this PR before merging it, it has nothing to do in repo.

**Comment by sorin-florea** [comment]
@lemeurherve Good catch, removed and updated `.gitignore`

File: src/main/java/hudson/plugins/gradle/injection/MavenBuildScanInjection.java [hunk 1]
```diff
@@ -26,6 +26,8 @@ public class MavenBuildScanInjection implements BuildScanInjection {
     private static final String GRADLE_ENTERPRISE_ALLOW_UNTRUSTED_SERVER_PROPERTY_KEY = "gradle.enterprise.allowUntrustedServer";
     private static final String GRADLE_SCAN_UPLOAD_IN_BACKGROUND_PROPERTY_KEY = "gradle.scan.uploadInBackground";
     private static final String MAVEN_EXT_CLASS_PATH_PROPERTY_KEY = "maven.ext.class.path";
+    private static final String GRADLE_SCAN_CUSTOM_VALUE_CI_AUTO_INJECTION = "scan.value.CI-auto-injection";
+    public static final String JENKINS = "Jenkins";
```
Comment (guylabs) [inline src/main/java/hudson/plugins/gradle/injection/MavenBuildScanInjection.java:30]:
```suggestion
    private static final String GRADLE_ENTERPRISE_ALLOW_UNTRUSTED_SERVER_PROPERTY_KEY = "gradle.enterprise.allowUntrustedServer";
    private static final String BUILD_SCAN_UPLOAD_IN_BACKGROUND_PROPERTY_KEY = "gradle.scan.uploadInBackground";
    private static final String MAVEN_EXT_CLASS_PATH_PROPERTY_KEY = "maven.ext.class.path";
    private static final String BUILD_SCAN_CI_AUTO_INJECTION_CUSTOM_VALUE_NAME = "scan.value.CI-auto-injection";
    public static final String BUILD_SCAN_CI_AUTO_INJECTION_CUSTOM_VALUE_VALUE = "Jenkins";
```

File: src/main/resources/hudson/plugins/gradle/injection/init-script.gradle [hunk 2]
```diff
@@ -48,7 +48,8 @@ def BUILD_SCAN_PLUGIN_CLASS = 'com.gradle.scan.plugin.BuildScanPlugin'
 def GRADLE_ENTERPRISE_PLUGIN_ID = 'com.gradle.enterprise'
 def GRADLE_ENTERPRISE_PLUGIN_CLASS = 'com.gradle.enterprise.gradleplugin.GradleEnterprisePlugin'
 def GRADLE_ENTERPRISE_EXTENSION_CLASS = 'com.gradle.enterprise.gradleplugin.GradleEnterpriseExtension'
-
+def CI_AUTO_INJECTION = 'CI-auto-injection'
+def JENKINS = 'Jenkins'
```
Comment (guylabs) [inline src/main/resources/hudson/plugins/gradle/injection/init-script.gradle:52]:
```suggestion
def CI_AUTO_INJECTION_CUSTOM_VALUE_NAME = 'CI-auto-injection'
def CI_AUTO_INJECTION_CUSTOM_VALUE_VALUE = 'Jenkins'
```

**Review by guylabs — CHANGES_REQUESTED** [review]
Some renamings of the variables.

**Review by facewindu — APPROVED** [review]

**Review by c00ler — APPROVED** [review]

**Review by guylabs — APPROVED** [review]

**Pull request merged**